llvm/test/tools/llvm-debuginfo-analyzer/WebAssembly/Inputs/definitions.h

```diff
@@ -0,0 +1,30 @@
+//-----------------------------------------------------------------------------
+// Definitions.
+//-----------------------------------------------------------------------------
+#ifndef SUITE_DEFINITIONS_H
+#define SUITE_DEFINITIONS_H
+
+#ifdef _MSC_VER
```

Comment:
since this is input just for testing, the compiler will always be clang and never MSVC. we could probably just write these definitions directly into the test. unless the test is shared with other compilers or platforms?

Reply:
All the test cases (`.cpp`) used by the `llvm-debuginfo-analyzer` for the WebAssembly are described in detail in the tool documentation.
We used GCC and MSVC to create binaries using those `.cpp` files that are used for the DWARF and COFF testing.
I included the `definitions.h` just to keep consistency between documentation and tests (binaries and sources).